src/theme/_typography.scss

```diff
@@ -4,14 +4,22 @@ $font-sans-serif: -apple-system, blinkmacsystemfont, "Segoe UI", "Roboto",
 $font-monospace: source-code-pro, menlo, monaco, consolas, "Courier New",
   monospace;
 
-@mixin body-text {
-  font-family: $font-sans-serif;
-  font-size: 16px;
-  line-height: 1.3;
+@mixin extra-large-text {
+  font-size: 34px;
 }
 
-@mixin headline-text {
-  font-family: $font-sans-serif;
+@mixin large-text {
   font-size: 24px;
-  line-height: 1.1;
+}
+
+@mixin medium-text {
+  font-size: 18px;
+}
+
+@mixin small-text {
+  font-size: 13px;
+}
+
+@mixin bold-text {
+  font-weight: 550;
```

Comment:
If I'm not mistaken these generalized types are where we should start creating the kss styleguide.

Reply:
new story for that?